tests/regression/results.csv

```diff
@@ -1,611 +1,611 @@
-silesia,                            level -3,                           compress cctx,                      6444663
-silesia,                            level -1,                           compress cctx,                      6178442
-silesia,                            level 0,                            compress cctx,                      4849491
-silesia,                            level 1,                            compress cctx,                      5313144
-silesia,                            level 3,                            compress cctx,                      4849491
-silesia,                            level 4,                            compress cctx,                      4786913
-silesia,                            level 5,                            compress cctx,                      4710178
-silesia,                            level 6,                            compress cctx,                      4659996
-silesia,                            level 7,                            compress cctx,                      4596234
-silesia,                            level 9,                            compress cctx,                      4543862
-silesia,                            level 13,                           compress cctx,                      4482073
-silesia,                            level 16,                           compress cctx,                      4377389
-silesia,                            level 19,                           compress cctx,                      4293262
-silesia,                            long distance mode,                 compress cctx,                      4849491
-silesia,                            multithreaded,                      compress cctx,                      4849491
-silesia,                            multithreaded long distance mode,   compress cctx,                      4849491
-silesia,                            small window log,                   compress cctx,                      7078156
-silesia,                            small hash log,                     compress cctx,                      6554898
-silesia,                            small chain log,                    compress cctx,                      4931093
-silesia,                            explicit params,                    compress cctx,                      4794609
-silesia,                            uncompressed literals,              compress cctx,                      4849491
-silesia,                            uncompressed literals optimal,      compress cctx,                      4293262
-silesia,                            huffman literals,                   compress cctx,                      6178442
-silesia,                            multithreaded with advanced params, compress cctx,                      4849491
+silesia.tar,                        level -5,                           compress simple,                    6738593
```

Comment:
It seems there is a general tiny compression ratio regression with this PR.
I presume this is because blocks generating < 4K symbols actually exist quite naturally in `silesia.tar`,
even with 128K full-size blocks,
and the removal of `-1` weight in these cases is slightly detrimental to compactness.

Reply:
Yeah, that is correct